### DISCUSSION TITLE
<title>About Upgrading To GitHub Pro

### DISCUSSION THREAD

**Discussion by Rockyread** [community/community#63720] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

GitHub Pro Pricing does not list follow information's:

1 Limit of Repository storage size
2 Limit of LFS storage size

**Comment by mehulrajdev** [comment, marked as answer]
Repository storage size: There's a soft cap of 100 GB for each repository.  If you exceed that, GitHub will send a friendly reminder to the repository owner. While pushing to oversized repositories is technically possible, it's not recommended since it might hamper performance.
LFS (Large File Storage) details: Individual LFS objects are capped at 2 GB. For the collective LFS storage, free accounts get about 1 GB of Git LFS storage. For GitHub Pro and above (like Team and Enterprise tiers), once you max out your included storage, you'd typically need to grab some additional data packs.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
